### ISSUE TITLE
Accuracy of the SiLU op is incorrect

### ISSUE THREAD

**Issue by zk1998** [mirage-project/mirage#99]
I found that the accuracy of the mirage graph generated by the gated_mlp example is completely mismatched. After testing, I found that the accuracy of the silu op could not be aligned. The reproduction code is:
```
import mirage as mi
import torch

if __name__ == "__main__":
    graph = mi.new_kernel_graph()
    X = graph.new_input(dims=(8, 4096), dtype=mi.float16)
    O = graph.silu(X)
    
    input_tensors = [
        torch.randn(8, 4096, dtype=torch.float16, device='cuda:0'),
    ]

    outputs = graph(inputs=input_tensors, outputs=[O])
    
    # ========= Correctness check =========
    silu = torch.nn.SiLU()
    output0_correct = silu(input_tensors[0])
    
    print('mirage output[0]:', outputs[0], sep='\n')
    print()
    print('correct output[0]:', output0_correct, sep='\n')
    
    # check the correctness of the output tensors
    print('Correctness of output[0]:', torch.allclose(outputs[0], output0_correct, atol=1e-3))
```
My result shows:
```
mirage output[0]:
tensor([[-0.0649,  0.2749,  0.1782,  ..., -0.0425, -1.0791, -0.5083],
        [-0.2474, -0.9648, -2.0488,  ...,  0.2686, -0.0774,  0.2732],
        [-1.1436,  0.2286,  0.2664,  ..., -0.1858,  0.2676, -0.8286],
        ...,
        [ 0.1395,  0.1825, -1.2383,  ...,  0.2644,  0.2130,  0.1720],
        [-0.6152,  0.2764,  0.2761,  ...,  0.2688,  0.1360,  0.1368],
        [-0.3640,  0.2686,  0.2167,  ...,  0.2749,  0.2336, -0.7979]],
       device='cuda:0', dtype=torch.float16)

correct output[0]:
tensor([[-0.0574,  1.1934,  0.2827,  ..., -0.0392, -0.2778, -0.2404],
        [-0.1639, -0.2783, -0.2134,  ...,  1.3369, -0.0670,  1.2344],
        [-0.2764,  0.4514,  0.6997,  ..., -0.1348,  1.3545, -0.2749],
        ...,
        [ 0.1949,  0.2939, -0.2732,  ...,  0.6792,  2.0527,  0.2666],
        [-0.2571,  1.1455,  1.1553,  ...,  1.3330,  0.1880,  0.1896],
        [-0.2058,  0.7256,  0.4023,  ...,  1.1934,  0.4744, -0.2734]],
       device='cuda:0', dtype=torch.float16)
Correctness of output[0]: False
```
The values are completely misaligned.

**Comment by xinhaoc** [comment]
Thanks for pointing out the issue @zk1998 !

**Comment by jiazhihao** [comment]
The issue should have been fixed by #100.
